acceptance_tests/__init__.py

```diff
@@ -9,6 +9,8 @@
 configure(logger_factory=LoggerFactory())
 
 if os.getenv('HEADLESS', 'True') == 'True':
+    browser = Browser('chrome', headless=True)
+elif os.getenv('HEADLESS', 'phantomjs') == 'phantomjs':
```

Comment:
Do we want to add support for a deprecated method of testing?

Reply:
Not adding just keep it for backwards compatibility in jenkins until we sort out the jenkins build

Reply:
Can the jenkins build not handle chrome?  I thought splinter brought drivers for all the browsers along for the ride?

Reply:
I don't think splinter pulls the drivers down, we've manually had to pull to driver down in the pipfile. Even if it did I don't think the driver actually equates to the actual browser, we'd need to install chrome on jenkins which isn't impossible just a breaking change that I don't want to introduce in this PR